src/ORM/Behavior/Translate/EavStrategy.php

```diff
@@ -386,7 +386,8 @@ protected function rowMapper(CollectionInterface $results, string $locale): Coll
             $row['_locale'] = $locale;
             if ($hydrated) {
                 /** @var \Cake\Datasource\EntityInterface $row */
-                $row->clean();
+                $row->setDirty($field, false);
```

Comment:
You'll need to add `$field = null` on line 368 and a null guard here for this to be type safe.

Reply:
Because this didn't change any tests, I would get the clean() was considered not test worthy. But, now we might need one to prevent regressions where unmodified fields are marked dirty.